src/cryptoadvance/specter/config.py

```diff
@@ -148,6 +152,10 @@ class TestConfig(BaseConfig):
     # API active by default in test-mode
     SPECTER_API_ACTIVE = _get_bool_env_var("SPECTER_API_ACTIVE", "True")
 
+    # See #1316 since Bitcoin v0.21.1 (not only) the importmulti-call takes longer than 10 seconds on cirrus
```

Comment:
If you can create a standalone example of this happening, please file a Github issue on https://github.com/bitcoin/bitcoin and tag me and @achow101 on it.